onnxruntime/contrib_ops/cuda/bert/attention.cc

```diff
@@ -140,7 +153,7 @@ Status Attention<T>::ComputeInternal(OpKernelContext* context) const {
   size_t workSpaceSize = GetAttentionWorkspaceSize(element_size,
                                                    batch_size,
                                                    num_heads_,
-                                                   head_size,
+                                                   qkv_head_size[0],
```

Comment:
Do we need change GetAttentionWorkspaceSize function as well? I did not see it in this PR.

Reply:
Yes - I just added the change that shouldn't over-allocate memory anymore